src/components/LoadingSpinner/index.jsx

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
 import { Icon } from '@edx/paragon';
+import PropTypes from 'prop-types';
 
-const LoadingSpinner = () => (
+const LoadingSpinner = props => (
   <div className="mx-auto text-center">
+    <div>
+      { props.message }
+    </div>
     <Icon
       id="spinner"
       className={['fa', 'fa-circle-o-notch', 'fa-spin', 'fa-3x', 'fa-fw']}
     />
   </div>
 );
 
+LoadingSpinner.defaultProps = {
+  message: 'Loading...',
```

Comment:
You might want the same trick used in LoadingMessage - null default and then do a `message ? message : loading...` in the html. That way someone can pass in null or empty string.

Why do we have LoadingMessage and LoadingSpinner? Feels a little overlapping.